src/+otp/Problem.m

```diff
@@ -149,7 +149,8 @@
     end
     
     methods (Access = protected)
-        % This method is called when either TimeSpan, Y0, or parameters are changed.  It should update F and other properties such as a Jacobian to reflect the changes.  This is effevtively an abstract function but not explicitly marked so in order to support Octave
+        % This method is called when either TimeSpan, Y0, or parameters are changed. It should
```

Comment:
What max line width are we defaulting to? I personally like 120 instead of 80 (we are not in the 70s anymore), but your call really.

Reply:
120 seems reasonable for the code. My only minor concern is when user calls `help` in a narrow command window it wrap doc text in ugly way.